src/commands/resumes/getResume.ts

```diff
@@ -0,0 +1,59 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+import { RESUME_EMBED_COLOUR } from '../../utils/embeds';
+
+class GetResumeCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'get-resume-command',
+      aliases: ['get-resume', 'resume'],
+      group: 'resumes',
+      memberName: 'get-resume',
+      description: 'Get an example resume!',
+      examples: [`${client.commandPrefix}resume web-dev 2`],
+      args: [
+        {
+          key: 'field',
```

Comment:
i need a way to know all possible fields, and how we spell it via reusing ``availableDomains`` from interview.ts to be consistent with other functions across codey.
This spelling should be consistent in the json too.

Maybe avail domains should be moved to somewhere with higher scope - choice is up to u.

Reply:
maybe another function tbh 💀

Reply:
what do you mean by another function XD I can move the `availableDomains` to a higher scope and modify `interview.ts` accordingly, that should be okay

Reply:
As a user, how would I know webdev is spelt ``web-dev``? There should be some sort of thing that gives information for the usage of this function - the complete list of ``availableDomains``

I may wanna see/know the entire stock, and see what we have, but actually nvm, no need.

Reply:
Wait I'm confused, do we need this then?

Reply:
Alright, I've made the fields the `availableDomains` from `components/interview.ts`.

I know discord.js v13 slash commands allows us to choose specific options for a field, so we can probably incorporate the availableDomains then.